stations.mss

```diff
@@ -20,13 +20,16 @@
     }
   }
 
-  [railway = 'station'][zoom >= 12] {
+  [railway = 'station'] {
```

Comment:
I did not test that (just look at z11), but it looks like that would render on any zoom level, not something we want...The limit should stay there, I think.

Reply:
Both data layers have minzoom 12, so probably it would not do any harm, but I prefer to have it visible in the styling code too.

Reply:
I changed the coding of zoom level. I checked and it works fine